app/src/main/java/com/mobilecoronatracker/ui/countrieslist/FragmentCountriesList.kt

```diff
@@ -88,8 +66,15 @@ class FragmentCountriesReports : Fragment() {
 
     private fun bindObservers() {
         viewModel.countryReports.observe(viewLifecycleOwner, Observer {
-            adapter.reports = it
+            adapter.deltas = it
             adapter.notifyDataSetChanged()
         })
+        viewModel.toggleCollapse.observe(viewLifecycleOwner, Observer {
```

Comment:
Which item collapsible_container will be called here?

Reply:
That was leftover, missed during refactoring. Removed.